libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -613,8 +641,7 @@ - (void) webView:(WKWebView *) webView decidePolicyForNavigationAction:(WKNaviga
             }
             shouldAllowRequest = NO;
         } else {
-            [self defaultWKNavigationHandling:webView decidePolicyForNavigationAction:navigationAction decisionHandler:decisionHandler];
```

Comment:
I don't know the background on `defaultWKNavigationHandling`, can the whole method go away now?

Reply:
Hmm not sure - I think my refactoring was too aggressive.

Reply:
Reverted that change with https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Hybrid/pull/256/commits/c02fde85e24c6db99f33e743fd2656f642bb8565